examples/register.rs

```diff
@@ -242,8 +242,8 @@ impl RegisterStateMachine {
 /// used in Raft. Feel encouraged to base yours of one of ours in these examples.
 impl state_machine::StateMachine for RegisterStateMachine {
 
-    /// `apply()` is called on when a client's `.propose()` is commited and reaches the state
-    /// machine. At this point it is durable and is going to be applied on at least half the nodes
+    /// `apply()` is called on when a client's `.propose()` is committed and reaches the state
+    /// machine. At this point it is durable and is going to be applied on a majority
```

Comment:
While you are fixing this, can you change the second sentence to read "At this point the proposal is durably stored on a majority of peers. The proposal will be applied to followers state machines after the next communication with the master."  The current prose is a not quite right.